components/searchBar/searchBar.tsx

```diff
@@ -1,19 +1,49 @@
-import React from 'react';
+import { countReset } from 'console';
+import React, { useState } from 'react';
+
+// Components
+import SearchSVG from '../svg/searchSVG';
 
 interface iSearchbar {
   placeholder: string;
+  value?: string;
+  setValue?: (e: any) => void;
 }
 
-const searchBar: React.FC<iSearchbar> = ({ placeholder }) => {
+const searchBar: React.FC<iSearchbar> = ({ placeholder, value, setValue }) => {
+  const handleChange = (e) => {
+    setValue(e.target.value);
+  };
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setValue(value);
+    setValue('');
```

Comment:
Why are we setting it empty after we replace the value with the same modificator? shouldn't be an effect a better option for this side effect?

Reply:
This reason is that after that we write on the input and submit, the input must is clean